src/cli/commands/ls.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const {print, rightpad} = require('../utils')
```

Comment:
Yes, that is changed now. Is there a place where the agreed code style is documented?
(The destructuring is not mentioned [js-project-guidelines.md](https://github.com/ipfs/community/blob/master/js-project-guidelines.md), nor in the [standard](https://github.com/standard/standard) code style)